web/src/App.vue

```diff
@@ -54,6 +55,13 @@
             <v-list-item-content>File Analysis</v-list-item-content>
           </v-list-item>
         </v-list>
+
+        <template v-slot:append>
+          <div class="pa-2" v-if="!isCollapsed">
```

Comment:
This does "hide" the pseudonymization feature a bit.

Maybe we could show an "incognito" symbol in the location of the toggle button if the navbar is collapsed:
![image](https://user-images.githubusercontent.com/3226995/171575016-4b80aba1-1c97-4fd7-826c-90aa8c13c7fa.png)

If the user would hover over this icon, the navbar will expand and the toggle button will become visible.